android/app/src/main/java/edu/berkeley/eecs/e_mission/ConfirmSectionListActivity.java

```diff
@@ -342,6 +342,10 @@ public void confirmAll(View view) {
 			}
 			statsHelper.storeMeasurement(getString(R.string.button_confirm_all_skipped), String.valueOf(skippedCount), confirmTs);
 		}
+        Intent intent = new Intent(ConfirmSectionListActivity.this,
```

Comment:
This is the substantive part of the change, right? Might be good to highlight that in the review (i.e. "We do this by relaunching the activity so that it refreshes).